t/APISIX.pm

```diff
@@ -730,6 +729,7 @@ _EOC_
         listen 1994 quic reuseport;
         listen 1994 ssl;
         http2 on;
+        http3 on;
```

Comment:
Why is this being added now? Why wasn't this added during the main feature PR?

Reply:
The `http3` directive default is `on`. And I forgot to add it in the main feature PR.